src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -796,15 +796,19 @@ private ImmutableArray<ParameterSymbol> ComputeParameters()
         internal sealed override void AddSynthesizedReturnTypeAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<CSharpAttributeData> attributes)
         {
             base.AddSynthesizedReturnTypeAttributes(moduleBuilder, ref attributes);
+            AddSynthesizedReturnTypeFlowAnalysisAttributes(this, _property, ref attributes);
+        }
 
-            var annotations = ReturnTypeFlowAnalysisAnnotations;
+        internal static void AddSynthesizedReturnTypeFlowAnalysisAttributes(MethodSymbol accessor, SourcePropertySymbolBase property, ref ArrayBuilder<CSharpAttributeData> attributes)
```

Comment:
>internal static void AddSynthesizedReturnTypeFlowAnalysisAttributes(MethodSymbol accessor, SourcePropertySymbolBase property, ref ArrayBuilder<CSharpAttributeData> attributes) [](http://example.com/codeflow?start=8&length=175)

Consider keeping this an instance member, simply extracting the code as is. The method would take just ```ref ArrayBuilder<CSharpAttributeData> attributes```.